setup_sunvdm.ps1

```diff
@@ -28,10 +30,29 @@ $height = [int]$args[1]
 $refresh_rate = [int]$args[2]
 $hdr = $args[3] -eq "true"
 $hdr_string = if($hdr) { "on" } else { "off" }
+
+Write-Output "hdr = $hdr_string"
+
+# + Choose the exact name of the Virtual Monitor to allow different versions without breaking the script.
+$vdd_name = $args[4]
+
+# + Add the feature of auto resolution/frame rate without the need of adding them manually in option.txt
+$option_to_check = "$width, $height, $refresh_rate"
+$option_file_path = "C:\IddSampleDriver\option.txt"
```

Comment:
Is this always the path to the option.txt file? I'm okay with assuming that (it's how the guide in their repo suggests doing the install), but maybe fail gracefully if it's not there. Also we should add something in our README about making sure the option.txt path is here.

Reply:
Checks added in the latest commit.